wsagent/agent/src/main/resources/org.eclipse.che.ws-agent.script.sh

```diff
@@ -55,8 +59,10 @@ MACHINE_TYPE=$(uname -m)
 
 mkdir -p ${CHE_DIR}
 ${SUDO} mkdir -p /projects
-${SUDO} sh -c "chown -R $(id -u -n) /projects"
-${SUDO} chmod 755 /projects
+if [ "${SUDO}" = "sudo -E" ]; then
```

Comment:
Here you need to check if the user has `sudoer` privileges.  Why don't you just use `if is_current_user_sudoer; then`. That would be more easy to read.